superset-frontend/src/filters/components/GroupBy/types.ts

```diff
@@ -40,7 +41,7 @@ export type PluginFilterGroupByProps = PluginFilterStylesProps & {
   data: DataRecord[];
   filterState: FilterState;
   formData: PluginFilterGroupByQueryFormData;
-  inputRef: RefObject<HTMLInputElement>;
+  inputRef: RefObject<SelectRef>;
```

Comment:
Is the `inputRef` used anywhere to access attributes from the `HTMLInputElement` directly?

Reply:
Yes. It's used here to automatically focus the filters:
https://github.com/apache/superset/blob/c959d92dd17499e3fb7a0f4f02f3781516f3d3e6/superset-frontend/src/dashboard/components/nativeFilters/FilterBar/FilterControls/FilterValue.tsx#L213